### PR TITLE
326 add button to reach setting page

### PR DESCRIPTION
**Pull request by julianhahn**
made a fiew minor changes to the overall look

### CODE REVIEW

**Review by anbraten — CHANGES_REQUESTED** [review]

File: packages/app/src/views/Home.vue [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
       <Button
         :to="{ name: 'add-tag' }"
         icon="fas fa-plus-square"
         text="Tag hinzufügen"
         :text-size="5"
-        class="py-2 mx-4 flex-grow"
+        both-center
+        class="mx-4 flex-grow"
       />
+      <Button :to="{ name: 'settings' }" icon="fas fa-cog" class="py-2 ml-2 mr-4 px-2" />
```
Comment (tcoro) [inline packages/app/src/views/Home.vue:12]:
```suggestion
      <Button :to="{ name: 'settings' }" icon="fas fa-cog" class="py-2 mx-2 px-2" />
```

File: packages/app/src/views/Home.vue [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-    <header class="flex py-5 justify-center text-4xl">
+    <header class="flex p-5 text-4xl">
       <Button
         :to="{ name: 'add-tag' }"
         icon="fas fa-plus-square"
         text="Tag hinzufügen"
         :text-size="5"
-        class="py-2 mx-4 flex-grow"
+        both-center
+        class="mx-4 flex-grow"
```
Comment (tcoro) [inline packages/app/src/views/Home.vue:10]:
```suggestion
        class="mx-1 flex-grow"
```

**Review by tcoro — CHANGES_REQUESTED** [review]
I suggest these changes, feels more relaxed to the eye

File: packages/app/src/views/Home.vue (hunk 1 above)
Comment (julianhahn) [reply packages/app/src/views/Home.vue]: okay,
fixed in 18e0a57672f8c08cc87053a75e68749a5cafd88e

File: packages/app/src/views/Home.vue (hunk 2 above)
Comment (julianhahn) [reply packages/app/src/views/Home.vue]: okay, 
fixed in 18e0a57672f8c08cc87053a75e68749a5cafd88e

**Review by tcoro — APPROVED** [review]
My name is Tom and I approve this :regional_indicator_p: :regional_indicator_r:

File: packages/app/src/views/AddTag.vue [hunk 3]
```diff
@@ -36,7 +35,11 @@
           :text-size="5"
           icon="fas fa-chevron-left"
           class="mr-4 px-4 py-2 text-center"
-          @click="activeStep--"
+          @click="
```
[the remark this answers is not in the document]
Comment (anbraten) [reply packages/app/src/views/AddTag.vue:38]: ? 
Could you try `activeStep+=1`

File: packages/app/src/views/Settings.vue [hunk 4]
```diff
@@ -68,9 +67,7 @@ export default defineComponent({
     };
 
     const loadUser = (): void => {
-      if (isAuthenticated) {
-        userEmail.value = user.value?.email;
-      }
+      if (isAuthenticated && user.value !== null) userEmail.value = user.value.email;
```
Comment (anbraten) [inline packages/app/src/views/Settings.vue:70]:
```suggestion
      if (isAuthenticated && user.value !== null) {
        userEmail.value = user.value.email;
      }
```

**Review by anbraten — CHANGES_REQUESTED** [review]

**Review by anbraten — APPROVED** [review]

**Review by JanEberlein — APPROVED** [review]
looks good

**Pull request merged**